### PR TITLE
Update waas-delivery-optimization.md

### PR DESCRIPTION
**Pull request by cmknox**
Add Edge browser support to content type table.

### CODE REVIEW

File: windows/deployment/update/waas-delivery-optimization.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ For information about setting up Delivery Optimization, including tips for the b
     - Office installations and updates
     - Xbox game pass games
     - MSIX apps (HTTP downloads only)
-    - Edge browser installations and updates
+    - Edge browser installs and updates
```
Comment (illfated) [inline windows/deployment/update/waas-delivery-optimization.md:68]: What is the intention behind changing "installations" to "installs"? Is it a different semantic (meaning) from the existing wording?

Comment (cmknox) [reply windows/deployment/update/waas-delivery-optimization.md:68]: Nope, just a preference.  Do you have an opinion?  I could go either way, but prefer `installs`.

Comment (illfated) [reply windows/deployment/update/waas-delivery-optimization.md:68]: Well, at first I was going to say that "installs" reads more often as a verb in the present tense than a noun, but I guess that is more the case for books and conservative writing. From what I have understood during the last year, it is the intention to keep the documentation less formal than traditional conservative books.

As long as "installs" is easily understood to be a noun in this case, it should be OK. I might not know the exact interpretation for some phrases, given that English is not my first language (although I read & write it on a near-native level).

Comment (cmknox) [reply windows/deployment/update/waas-delivery-optimization.md:68]: Ok, let's go with it and change it later, if needed.

**Review by illfated — APPROVED** [review]
Looks quite OK to me.

#sign-off

**Review by illfated — APPROVED** [review]
Looks like a good addition to have the added content linked to its relevant blog post.

**Pull request merged**